mlflow/transformers/__init__.py

```diff
@@ -2928,6 +3005,19 @@ def _format_prompt_template(self, input_data):
             "Prompt templating is only supported for data of type str or List[str]. "
         )
 
+    def _postprocess_output_for_inference_task(self, data, output):
+        """
+        Wrap output data with usage information according to the MLflow inference task.
```

Comment:
nit: (1) Could you add input/output example in doc string? (2) Maybe we can move this to `inference_utils.py`.